### PR TITLE
Stg

### PR DESCRIPTION
**Pull request by kpslp**

### CODE REVIEW

File: docs/whats-new/staging-release-notes.md [hunk 1]
```diff
@@ -2,6 +2,50 @@
...
+We have updated TechPass portal with a new search bar feature that allows users to search for specific applications. Additionally, tenants can also use the search bar to search for applications and OTP Credentials.
+
+</details>
+
+**Changes** 
+
+<details>
+<summary style="font-size:20px;font-weight:bold"> Updated tenants' restriction on TechBiz application role assignment & groups management in TechPass </summary>
+
+Previously, tenants were not allowed to manage their applications' role assignments on TechPass if they had onboarded to TechBiz. However, we understand that some applications may not be on TechBiz, so we have updated the restriction to be applied at the application-level instead.
```
Comment (JLeeChan) [inline docs/whats-new/staging-release-notes.md:23]: It took me some time to understand this:
**Previously, tenants were not allowed to manage their applications' role assignments on TechPass if they had onboarded to TechBiz. However, we understand that some applications may not be on TechBiz**

Perhaps we explain more clearly the differencedifference between tenant-wide level vs application level TechBiz onboarding.

File: docs/whats-new/staging-release-notes.md [hunk 2]
```diff
@@ -2,6 +2,50 @@
...
+</details>
+
+**Changes** 
+
+<details>
+<summary style="font-size:20px;font-weight:bold"> Updated tenants' restriction on TechBiz application role assignment & groups management in TechPass </summary>
+
+Previously, tenants were not allowed to manage their applications' role assignments on TechPass if they had onboarded to TechBiz. However, we understand that some applications may not be on TechBiz, so we have updated the restriction to be applied at the application-level instead. 
+
+Additionally, TechBiz-created groups will no longer be displayed on the TechPass portal, and tenants will not be able to call certain Automation APIs for these groups. To help tenants exclude TechBiz-created groups from their results, we have introduced a new *excludeOnTechBiz* query parameter for the **List Groups** Automation API. These changes are designed to streamline the management of application role assignments and groups for TechPass tenants.
```
Comment (JLeeChan) [inline docs/whats-new/staging-release-notes.md:25]: What is the default value of _excludeOnTechBiz_? I would think the value should be **true** by default.

Comment (yqyeoh) [reply docs/whats-new/staging-release-notes.md:25]: default is `false` currently , will change to `true`

Comment (yqyeoh) [reply docs/whats-new/staging-release-notes.md:25]: actually would it be better to change the param to `includeOnTechBiz` & have it `false` by default?

Comment (kpslp) [reply docs/whats-new/staging-release-notes.md:25]: ok let me know what i need to change to

File: docs/whats-new/staging-release-notes.md (hunk 1 above)
Comment (kpslp) [reply docs/whats-new/staging-release-notes.md:23]: @yqyeoh could you quickly provide me the difference

Comment (yqyeoh) [reply docs/whats-new/staging-release-notes.md:23]: by identifying TechBiz onboarding at the tenant level, we are restricting tenants from managing the app role assignments for ALL of their applications on TechPass (as we are expecting they should all be managed on TechBiz); by moving down to application level, we are imposing the restriction per application (i.e. on TechPass, tenants cannot manage the app role assignments for applications that are on TechBiz; whereas they will still be able to manage the app role assignments for applications that are not on TechBiz)

File: docs/whats-new/staging-release-notes.md (hunk 2 above)
Comment (yqyeoh) [reply docs/whats-new/staging-release-notes.md:25]: no need to change the release notes, i'll change the implementation in the next release

Comment (JLeeChan) [reply docs/whats-new/staging-release-notes.md:25]: yes, agree with includeOnTechBiz and set the default to false

Comment (JLeeChan) [reply docs/whats-new/staging-release-notes.md:25]: this way the portal and API have consistent default behaviour

File: docs/whats-new/staging-release-notes.md (hunk 1 above)
Comment (kpslp) [reply docs/whats-new/staging-release-notes.md:23]: updated in new commit

**Review by JLeeChan — APPROVED** [review]

**Pull request merged**